crates/uv-interpreter/src/interpreter.rs

```diff
@@ -284,32 +306,103 @@ impl Interpreter {
     pub fn sys_executable(&self) -> &Path {
         &self.sys_executable
     }
+
+    /// Return the `purelib` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn purelib(&self) -> &Path {
+        &self.sysconfig.purelib
+    }
+
+    /// Return the `platlib` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn platlib(&self) -> &Path {
+        &self.sysconfig.platlib
+    }
+
+    /// Return the `scripts` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn scripts(&self) -> &Path {
+        &self.sysconfig.scripts
+    }
+
+    /// Return the `data` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn data(&self) -> &Path {
+        &self.sysconfig.data
+    }
+
+    /// Return the `include` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn include(&self) -> &Path {
+        &self.sysconfig.include
+    }
+
+    /// Return the `stdlib` path for this Python interpreter, as returned by `sysconfig.get_paths()`.
+    pub fn stdlib(&self) -> &Path {
+        &self.sysconfig.stdlib
+    }
+
+    /// Return the [`Target`] environment used to install wheels into this interpreter.
+    pub fn target(&self) -> Target {
+        Target {
+            python_version: self.python_tuple(),
+            sys_executable: self.sys_executable().to_path_buf(),
+            purelib: self.purelib().to_path_buf(),
+            platlib: self.platlib().to_path_buf(),
+            scripts: self.scripts().to_path_buf(),
+            data: self.data().to_path_buf(),
+            // Detect if this is a PEP 405-compliant virtual environment.
+            // See: https://github.com/pypa/pip/blob/0ad4c94be74cc24874c6feb5bb3c2152c398a18e/src/pip/_internal/utils/virtualenv.py#L14.
+            include: if self.prefix == self.base_prefix {
+                // If the interpreter is a venv, then the `include` directory has a different structure.
+                // See: https://github.com/pypa/pip/blob/0ad4c94be74cc24874c6feb5bb3c2152c398a18e/src/pip/_internal/locations/_sysconfig.py#L172
+                self.prefix.join("include").join("site").join(format!(
+                    "python{}.{}",
+                    self.python_major(),
+                    self.python_minor()
+                ))
+            } else {
+                self.include().to_path_buf()
+            },
+        }
+    }
 }
 
+/// The installation paths returned by `sysconfig.get_paths()`.
+///
+/// See: <https://docs.python.org/3.12/library/sysconfig.html#installation-paths>
 #[derive(Debug, Deserialize, Serialize, Clone)]
-pub(crate) struct InterpreterInfo {
-    pub(crate) markers: MarkerEnvironment,
-    pub(crate) base_exec_prefix: PathBuf,
-    pub(crate) base_prefix: PathBuf,
-    pub(crate) stdlib: PathBuf,
-    pub(crate) sys_executable: PathBuf,
+struct Sysconfig {
```

Comment:
I'd go with `SysconfigInstallationPaths` or `SysconfigInstallationPaths`

Reply:
I will do `SysconfigPaths`, since that matches the name of the method.